src/js/components/projects/detail.tsx

```diff
@@ -84,7 +84,7 @@ const ProjectDetail = (props: RouteComponentProps) => {
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
   const usersAssignedToTasks = new Set<string>();
-  (tasks || []).forEach((task) => {
+  (tasks || []).forEach(task => {
```

Comment:
I've switched to using a workspace so I think I need to change some of the formatting settings.

Reply:
Hm... Looks like a `prettier` change -- are you sure you have the latest deps installed via `./derrick build`, or just `yarn` and `yarn lint:js` outside the container?

Reply:
I _think_ so, but I'll that.